app/helpers/users.js

```diff
@@ -13,3 +15,14 @@ exports.encryptPayload = async payload => {
     throw errors.defaultError('Something went wrong trying to encrypt password');
   }
 };
+
+exports.authenticateUser = email => {
```

Comment:
This should be more generic, like generateToken and receive a generic param like `payload`